File: web3rpc/rpc-specs/code-samples/javascript/kaia/miscellaneous/sha3.js
const { JsonRpcProvider } = require("@kaiachain/ethers-ext");
(() => {
  const provider = new JsonRpcProvider("https://public-en-kairos.node.kaia.io");

  const data = "0x11223344";

  provider.kaia
    .sha3(data, {}, (err, data, response) => {})
    .then((data) => {
      console.log(data);
    });
})();

File: web3rpc/rpc-specs/code-samples/python/kaia/miscellaneous/sha3.py
from web3 import Web3
from web3py_ext import extend

host = "https://public-en-kairos.node.kaia.io"

hashData = "0x11223344"

w3 = Web3(Web3.HTTPProvider(host))
kaia_response = w3.kaia.sha3(hashData)

print(kaia_response)
